docs/industrial_optimization/productizing_apps.rst

```diff
@@ -0,0 +1,171 @@
+.. _opt_deploying_quantum_apps_prod:
+
+=======================
+Deploying in Production
+=======================
+
+This section provides insight into designing and preparing a hybrid application
+to be successfully deployed in a production environment. Hybrid applications
+can be easy to integrate with a business's technology stack, in part, because
+such applications typically run batch-based, asynchronous operations, submitting
+problems to and receiving results from hybrid solvers in the Leap service.
+
+.. _quantum_apps:
+
+Application
```

Comment:
This title doesn't indicate what the content of this section is about. 

- The first paragraph below it gives an overview of the production environment and its elements. 
- The remainder describes the use of a hybrid application over time

If you separate those two into two sections, you'll be able to give more meaningful titles and users can skip what they already know without missing info that might be helpful to them.

Something like "Typical Production Environments" and "Operations of a Hybrid Application" or some such.